lib/Common/PlatformAgnostic/AssemblyCommon.h

```diff
@@ -0,0 +1,30 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+#ifndef _WIN32
+
+extern "C" void __register_frame(const void* ehframe);
+extern "C" void __deregister_frame(const void* ehframe);
+
+#if defined(_AMD64_) && !defined(DISABLE_JIT)
```

Comment:
Thanks @digitalinfinity for explanation, now I understand. `!defined(DISABLE_JIT)` here is fine to me.